tools/ci/tc/tasks/test.yml

```diff
@@ -230,6 +230,7 @@ tasks:
               --
               --channel=${vars.channel}
               --verify
+              --verify-no-chaos-mode
```

Comment:
Please make sure to document the reasoning for this either in the yml or at least the commit message, to give future-us an understanding of why we made this change :)